### PR TITLE
fix(line-height): this ensures the text is dead center on the icons w…

### PR DESCRIPTION
**Pull request by mhewson**
…ithin rating, this update will

### Design:
- [ ] Reviewed with designer and meets expectations

### Cross-browser testing:
- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari
- [ ] IE11
- [ ] iOS
- [ ] Android

### Visual regression testing:
- [ ] Added/updated backstop tests
- [ ] Passes with existing reference images (or failed as expected)

### Unit testing:
- [ ] Sufficient unit test coverage (see unit test best practices for ideas)

### A11y:
- [ ] Meets WCAG AA standards

### Documentation:
- [ ] API docs created/updated
- [ ] Examples created/updated

### CODE REVIEW

**Comment by lancetharper** [comment]
This "works" for the current typography because it sets the line-height to be greater than the height of the icons. If/when typography changes or icon size changes this may not work anymore.

If we're ok with having magic numbers like this, then I guess we just accept that...but this is still just a band-aid and doesn't fix the underlying issue

**Comment by darintrimillos** [comment]
> This "works" for the current typography because it sets the line-height to be greater than the height of the icons. If/when typography changes or icon size changes this may not work anymore.
> 
> If we're ok with having magic numbers like this, then I guess we just accept that...but this is still just a band-aid and doesn't fix the underlying issue

I'm OK with the band-aid because it seems like it might be a while before we have a better fix figured out.

File: src/components/rating/styles/vars/CdrRating.vars.scss [hunk 1]
```diff
@@ -8,12 +8,15 @@ $rating-number-decoration: $cdr-color-border-secondary-lightmode;
 
 @mixin cdr-rating-base-text() {
   @include cdr-text-utility-200;
+  line-height: 1.8;
```
Comment (lancetharper) [inline src/components/rating/styles/vars/CdrRating.vars.scss:11]: This is the only change that actually affects the height afaict. The others just set the line-height equal to the icon height which doesn't change anything alignment-wise that I can see

Comment (mhewson) [reply src/components/rating/styles/vars/CdrRating.vars.scss:11]: true for the fonts demo'ed but having the heights of each exactly equal seems like the best way to ensure that nothing ever moves them out of alignment

**Comment by cowills** [comment]
@mhewson is this ready to review and merge?

**Comment by mhewson** [comment]
@cowills,  I'm planning on going over this s at review -- fixing this has other impacts to alignment with other text that I want design to be aware of

**Comment by cowills** [comment]
@mhewson can we handle the a11y fixes in a separate PR? i don't want that to block the patch release

**Pull request closed without merging**